blimp-live-widgets/src/components/DateComponent/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+import { updateDate } from './utils';
+
+export default class DateComponent extends Component {
+
+  
+  constructor(props) {
+    var currentDate = new Date()
+    super(props);
+    
+    const date = this.initializeDate(props.gmtOffset);
```

Comment:
Can u check that this is set up correctly - like is the prop being set correctly?